flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/parse/ClearOperationParseStrategy.java

```diff
@@ -29,7 +29,7 @@
     static final ClearOperationParseStrategy INSTANCE = new ClearOperationParseStrategy();
 
     private ClearOperationParseStrategy() {
-        super(Pattern.compile("CLEAR", DEFAULT_PATTERN_FLAGS));
```

Comment:
Why this change?

Reply:
it is because that in this pr, we didn't truncate ';' in sql client, parser need to support commands in both 'clear' and 'clear;'